packages/@aws-cdk/aws-apigatewayv2/lib/http/api.ts

```diff
@@ -353,6 +372,9 @@ export class HttpApi extends HttpApiBase {
 
   private readonly _apiEndpoint: string;
 
+  private readonly authorizer?: IHttpRouteAuthorizer;
+  private readonly authorizationScopes?: string[];
```

Comment:
Just to be clear. Can we call this `defaultAuthorizer` and `defaultAuthorizationScopes` as well.

Reply:
Makes sense 👍🏾